src/components/Buttons/LargeOutlinedWIcon/LargeOutlinedWIconButtonStyled.js

```diff
@@ -2,29 +2,27 @@ import { device } from '@/constants/mediaQueries';
 import styled from 'styled-components';
 
 export const LargeOutlinedWIconButtonStyled = styled.button`
-width: 148px;
-height: 44px;
-padding: 12px 24px;
-border-radius: 8px;
-border: 1px solid var(--color-brand-blue);
-color: var(--color-grey-8);
-font-family: "Roboto", sans-serif;
-font-weight: 700;
-line-height: 19.2px;
-text-align: center;
-font-size: 14px;
-background-color: transparent;
-cursor: pointer;
-white-space: nowrap;
-    ${({ $mobilestyles }) => $mobilestyles};
+  width: 148px;
+  height: 44px;
+  padding: 12px 24px;
+  border-radius: 8px;
+  border: 1px solid var(--color-brand-blue);
+  color: var(--color-grey-8);
+  font-family: 'Roboto', sans-serif;
+  font-weight: 700;
+  line-height: 19.2px;
+  text-align: center;
+  font-size: 14px;
```

Comment:
Maybe we have to move all fonts properties fo the buttons to the Global styles?

Reply:
done